### ISSUE TITLE
Выгрузка в csv

### ISSUE THREAD

**Issue by AlekseySmorshchok** [GloomlySeth/Zapravka#14]
Если будет отсутствовать информация по заданным критериям, то выбивать сообщение пусто или скачивать пустой файл?

**Comment by GloomlySeth** [comment]
выводи сообщение: "Данные по запрашиваемому временному интервалу отсутствуют"

**Comment by AlekseySmorshchok** [comment]
насчет дат в выгрузке, тоже подставлять как в графиках?

**Comment by GloomlySeth** [comment]
да. Выгрузку делаем в привязке к интервалу выбранному на графиках.

по крайней мере пока

**Comment by GloomlySeth** [comment]
Если не долго можно список регионов с количеством АЗС в них

**Comment by AlekseySmorshchok** [comment]
просто некликабельный список сбоку. как первоначальный вид дерева?

**Comment by AlekseySmorshchok** [comment]
если поменяем в выгрузке дату , в графиках старая дата будет или новая?

**Comment by GloomlySeth** [comment]
2 варианта выгрузки
 Вариант 1 - выгружаем только то, что попадает в установленные даты. То есть то, что видим на графике. 
Вариант 2 - выгружаем всё,что есть. 
на сколько я понимаю, должно быть 2 кнопки. Либо нужно знать, когда начался съём. Тогда годиться только вариант 1.

**Comment by GloomlySeth** [comment]
а долго сделать, чтоб десятичные части цифр отделялись запятой?

**Comment by AlekseySmorshchok** [comment]
не сложно сделал

**Comment by GloomlySeth** [comment]
а у меня в двух редакторах точки остались (см. скрин)

забыл сервер обновить? 

![image](https://user-images.githubusercontent.com/18662965/52669099-569e7a00-2f26-11e9-942c-29b114548001.png)

**Comment by AlekseySmorshchok** [comment]
я не заливал еще, ждал ваш комментарий
